### ISSUE TITLE
Is it possible to compute gradients using BrainScript?

### ISSUE THREAD

**Issue by GCornbake** [Microsoft/CNTK#2852]
Hi,
    I want to show the receptive field of one unit in my output after training by means of computing the gradients through backpropagation(i.e. setting the output unit to 1 and then computing the gradients of the all the pixels in input image w.r.t. this output unit). I don't know is that feasible by using BrainScript and if yes, could someone tell me how to do that please. Thanks in advance.

**Comment by KeDengMS** [comment]
AFAIK BrainScript does not support this. You need to use Python or C++ APIs for this feature.
